index.html

```diff
@@ -236,6 +236,32 @@ <h4 class="connect">LET'S CONNECT</h4>
       </div>
 
     </section>
+    <section class="form-container">
+      <div class="form-min">
+      <div class="form-des">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      </div>
+      <form action="POST">
+      <ul>
+        <li>
+          <input type="text" name="name" id="user-name" placeholder="Enter Your Name">
+        </li>
+        <li>
+          <input type="email" name="email" id="email-address" placeholder="Please Enter your E-mail">
+        </li>
+        <li>
+          <textarea name="message" id="msg" cols="30" rows="10" placeholder="Enter Your Message Here"></textarea>
+        </li>
+        <li>
+          <button type="button">Get In Touch</button>

```

Comment:
Your type for the button should be `submit`.  Or you can leave it blank because the type by default is `submit`. Kindly make changes to the button.